handshake/server_config_client_test.go

```diff
@@ -184,6 +184,12 @@ var _ = Describe("Server Config", func() {
 				Expect(err).To(MatchError("CryptoInvalidValueLength: PUBS"))
 			})
 
+			It("rejects PUBS values that have a zero length", func() {
+				tagMap[TagPUBS] = bytes.Repeat([]byte{0}, 100) // completely wrong length
+				err := scfg.parseValues(tagMap)
+				Expect(err).To(MatchError("CryptoInvalidValueLength: PUBS"))
+			})
+
```

Comment:
This test is good, but I'd also like to have one where the C255 pubs is not at index 0. I'm fine with having a dummy value before that (e.g. 0x01, 0x00, 0x00, 0x00).